lib/quicksilver-config-client.js

```diff
@@ -51,6 +58,21 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration path (qsConfigPath) value defined in .env as QS_CONFIG_PATH
+   * is a valid path.
+   */
+  configFileFound() {
+    return new Promise((resolve) => {
+      fs.lstat(this.qsConfigPath, (err) => {
```

Comment:
You also need to check that this is a file. See [`stats.isFile()`](https://nodejs.org/api/fs.html#fs_class_fs_stats).